ext/opentelemetry-ext-grpc/src/opentelemetry/ext/grpc/__init__.py

```diff
@@ -21,7 +21,7 @@
 from opentelemetry.ext.grpc.version import __version__
 
 
-def client_interceptor(tracer_provider=None):
+def client_interceptor(tracer_provider=None, meter=None):
```

Comment:
How is `client_interceptor` actually used? Is this instantiated by the user? How does this relate to [#788]?

Reply:
```
interceptor = client_interceptor(meter=meter)
self.channel = intercept_channel(
    grpc.insecure_channel("localhost:25565"), interceptor
)
```

I will add a meter parameter to `_instrument` if #788 gets merged first (although.. passing a meter makes uninstrumentation difficult. maybe exporter + interval as params is actually a better choice in that case?

Reply:
An exporter + interval would make more. We can instantiate a specific meter for `grpc` metrics under the hood (unique name) and start the pipeline implicitly if an exporter is passed in.

Reply:
switched to using exporter + interval which will do for now, but I still think there needs to be a proper solution to this that doesn't require passing two things to every single instrumentation